charts/README.md

```diff
@@ -0,0 +1,78 @@
+# Installation with Helm 3
+
+Quick start instructions for the setup and configuration of SMB CSI driver using Helm.
+
+## Prerequisites
+
+1. [install Helm Client 3.0+ ](https://helm.sh/docs/intro/quickstart/#install-helm)
+
+## Install latest CSI Driver via `helm install`
+
+```console
+$ cd $GOPATH/src/sigs.k8s.io/csi-driver-smb/charts/latest
```

Comment:
I think this path is incorrect. I was unsure of the correct path.

Reply:
use `$GOPATH/src/github.com/kubernetes-csi/csi-driver-smb/charts/latest`

Reply:
Thanks. I have updated it :)